src/phoenix/server/app.py

```diff
@@ -604,6 +602,7 @@ def create_app(
     secret: Optional[str] = None,
     scaffolder_config: Optional[ScaffolderConfig] = None,
 ) -> FastAPI:
+    logger.info(f"Server umap params: {umap_params}")
```

Comment:
Probably okay to nix. Probably more like than not to confuse end users of tracing.